tests/nmstate_suite_test.go

```diff
@@ -64,3 +71,83 @@ func init() {
 	manifests = flag.String("manifests", "", "path to manifests to test")
 	flag.Parse()
 }
+
+func FindInterfaceInfo(name string, interfaces []nmstatev1.InterfaceInfo) (nmstatev1.InterfaceInfo, bool) {
```

Comment:
can we move these under a separate utils module?

Reply:
done